Set-Game/Set-Game/CardView.swift

```diff
@@ -0,0 +1,274 @@
+        return path
+    }
+    
+    private func squiggle(upperCenterPoint: CGPoint) -> UIBezierPath {
+        let squiggleWidth = frame.width / 10
+        let lowerCenterPoint = CGPoint(x: upperCenterPoint.x, y: upperCenterPoint.y + squiggleWidth)
+        let upperLeftPoint = CGPoint(x: upperCenterPoint.x - squiggleWidth*2, y: upperCenterPoint.y)
+        let upperRightPoint = CGPoint(x: upperCenterPoint.x + squiggleWidth*2, y: upperCenterPoint.y)
+        let lowerLeftPoint = CGPoint(x: lowerCenterPoint.x - squiggleWidth*2, y: lowerCenterPoint.y)
+        let lowerRightPoint = CGPoint(x: lowerCenterPoint.x + squiggleWidth*2, y: lowerCenterPoint.y)
+        
+        let halfSquiggleWidth = squiggleWidth / 1.5
+        let controlPoint1 = CGPoint(x: upperLeftPoint.x - halfSquiggleWidth, y: upperLeftPoint.y - halfSquiggleWidth)
+        let controlPoint2 = CGPoint(x: upperCenterPoint.x - halfSquiggleWidth, y: upperCenterPoint.y - halfSquiggleWidth)
+        let controlPoint3 = CGPoint(x: upperCenterPoint.x + halfSquiggleWidth, y: upperCenterPoint.y + halfSquiggleWidth)
+        let controlPoint4 = CGPoint(x: upperRightPoint.x, y: upperRightPoint.y + halfSquiggleWidth)
+        let controlPoint5 = CGPoint(x: lowerRightPoint.x + halfSquiggleWidth, y: lowerRightPoint.y + halfSquiggleWidth)
+        let controlPoint6 = CGPoint(x: lowerCenterPoint.x + halfSquiggleWidth, y: lowerCenterPoint.y + halfSquiggleWidth)
+        let controlPoint7 = CGPoint(x: lowerCenterPoint.x - halfSquiggleWidth, y: lowerCenterPoint.y - halfSquiggleWidth)
+        let controlPoint8 = CGPoint(x: lowerLeftPoint.x, y: lowerLeftPoint.y - halfSquiggleWidth)
+        
+        let path = UIBezierPath()
+        
+        path.move(to: lowerLeftPoint)
+        path.addCurve(to: upperCenterPoint, controlPoint1: controlPoint1, controlPoint2: controlPoint2)
+        path.addCurve(to: upperRightPoint, controlPoint1: controlPoint3, controlPoint2: controlPoint4)
+        path.addCurve(to: lowerCenterPoint, controlPoint1: controlPoint5, controlPoint2: controlPoint6)
+        path.addCurve(to: lowerLeftPoint, controlPoint1: controlPoint7, controlPoint2: controlPoint8)
+        
+        return path
+    }
+    
+    private func getCardColor(cardProperties: CardProperties) -> UIColor {
+        switch cardProperties.color {
+        case "red":
+            return #colorLiteral(red: 0.8035931587, green: 0.3512506187, blue: 0.459513247, alpha: 1)
+        case "green":
+            return #colorLiteral(red: 0.2272136807, green: 0.6645996571, blue: 0.6210204363, alpha: 1)
+        default:
+            return #colorLiteral(red: 0.9529411793, green: 0.6862745285, blue: 0.1333333403, alpha: 1)
+        }
+    }
+    
+    private func getSymbolOriginPoints(cardProperties: CardProperties) -> [CGPoint] {
+        var symbolOriginPoints = [CGPoint]()
+        var divider = CGFloat()
+        if (cardProperties.symbol == "oval") {
+            divider = 3.0
+        } else { divider = 2.0}
```

Comment:
We don't usually write an else statement on the same line. It should be:
else {
    divider = 2.0
}
Or, for this case and other simple cases like this you could use a conditional operator like:
divider = cardProperties.symbol == "oval" ? 3.0 : 2.0